charts/invenio/values.yaml

```diff
@@ -559,6 +559,132 @@ workerBeat:
     ##
     strategy: null
 
+## @section Terminal
+## Pods for runnning maintenance tasks.
+terminal:
+  ## @param terminal.enabled Enable the terminal deployment.
+  ##
+  enabled: true
```

Comment:
Is this ever used?

Also: Does it make sense to enable by default? Seems like an unnecessary security risk and resource utilization that should be up to people to opt-in for.